contracts/vortex/interfaces/ICarbonVortex.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: BUSL-1.1
-pragma solidity ^0.8.0;
+pragma solidity 0.8.19;
```

Comment:
In general, it's better to have a flexible `pragma` in interfaces (and common definitions), so that we won't force external integrations to use a specific solidity version.